docs/efun/index.md

```diff
@@ -199,11 +199,14 @@ title: EFUN
 * [has_gmcp](interactive/has_gmcp.html)
 * [has_mxp](interactive/has_mxp.html)
 * [has_zmp](interactive/has_zmp.html)
+* [has_msp](interactive/has_msp.html)
+* [has_msdp](interactive/has_masdp.html)
```

Comment:
Shouldn't this be `has_msdp.html`?

Reply:
yeah it should i miss it, i probably hit the a/s key at the same time and didnt think to double check the link assuming it was right, fixing it now

Reply:
the index are actually auto generated, just ran `./update_index.sh`